portfolio/src/main/webapp/style.css

```diff
@@ -8,27 +8,26 @@
 }
 
 .logo {
-    font-family: 'Aclonica';
-    margin-left: 42%;
-    font-size: auto;
-    
+  font-family: "Aclonica";
+  font-size: auto;
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties): Use shorthand properties where possible.

This page has a list of shorthand properties:
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties

Maybe fix in a follow-up PR?